### DISCUSSION TITLE
Please rename all processes to xcha_* (not chia_*)

### DISCUSSION THREAD

**Discussion by rwarren** [Chia-Network/chia-blockchain#8323] [Ideas]
Right now all processes are `chia_*`. They should be renamed to `xcha_*`.

**Comment by loppefaaret** [comment]
why would we call them `xcha` ? im not sure i follow your thinking here ?

**Comment by BrandtH22** [comment]
@rwarren , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
